### PR TITLE
CI fixes

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by mdellweg** [comment]
if ansible-core is installed, it returns `ansible [core 2.11.0b1]` on `ansible --version` and ansible-lint is not able to cope with this.

Failing here:
https://github.com/ansible-community/ansible-lint/blob/master/src/ansiblelint/config.py#L97

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@ cryptography<3.1; python_version < '3.6'
 -r requirements.txt
 pylint==2.6.0; python_version >= '3.6'
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.validate-modules.txt
-molecule[docker]
+molecule[docker]>=3.2.4
```
Comment (ehelms) [inline requirements-dev.txt]: Seems to not like this.

**Comment by ehelms** [comment]
I guess we need to add yamllint as an explicit requirement?

File: requirements-dev.txt (hunk 1 above)
Comment (evgeni) [reply requirements-dev.txt]: "python 2.7", yeah

Comment (ehelms) [reply requirements-dev.txt]: I don't think that version supports Python 2.7. Seems like it was last supported here https://pypi.org/project/molecule/3.0.2.1/. So gotta keep it the way it was or drop Py 2.7

**Review by ehelms — APPROVED** [review]
Katello repos failures is as Katello repo failures does.

**Pull request merged**